tests/src/main/scala/tests.scala

```diff
@@ -176,5 +176,13 @@ object Tests extends TestApp {
     test("serialize a value class") {
       Show.gen[Length].show(new Length(100))
     }.assert(_ == "100")
+
```

Comment:
This test does not actually get run when you call `sbt tests/run`! The reason is that the test library only runs tests which are in the execution path, and as `ParentClass` is not instantiated anywhere, it doesn't get run! (NB. This is not explained *anywhere*, so can be completely forgiven! ;) The test ran successfully when I tried it.)

Reply:
Actually, I just tested a refactored version of this test on the current release, `0.6.1`, and it worked fine, because I was misunderstanding exactly what you were fixing. ;) After a bit more experimentation, I realise that the interesting case is where the macro expands within another class. So, it should be sufficient to just instantiate `ParentClass` in the main body of the tests to get the test to a) run, and b) test the thing that's been fixed!